### PR TITLE
Art fixes

### PR DESCRIPTION
**Pull request by erasche**
I don't know why this wasn't a separate package in the original #99

### CODE REVIEW

File: packages/package_art_2014_11_03/tool_dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      <actions>
+        <action type="download_by_url">http://www.niehs.nih.gov/research/resources/assets/docs/artsrcvanillaicecream031114linuxtgz.tgz</action>
+        <action type="set_environment_for_install">
+          <repository name="package_gsl_1_16" owner="iuc">
+            <package name="gsl" version="1.16"/>
+          </repository>
+        </action>
+        <action type="autoconf"></action>
+        <action type="set_environment">
+          <environment_variable action="prepend_to" name="PATH">$INSTALL_DIR/bin/</environment_variable>
```
Comment (bgruening) [inline packages/package_art_2014_11_03/tool_dependencies.xml]: can you add ART_ROOT_PATH

**Comment by bgruening** [comment]
LGTM!

**Comment by erasche** [comment]
Should be ready to go. :)

**Comment by erasche** [comment]
@Unode this was actually a fix for you, when I saw you mention using another website to generate fastq data. I was going to link you the `art` software in the MTS but discovered that it wasn't there/wasn't working properly. Now it's fixed.

https://toolshed.g2.bx.psu.edu/view/iuc/art/a12ce5668966 is available for the next time you need to generate fastq data.

**Comment by Unode** [comment]
Thanks @erasche, so nice of you.
Letting @nsoranzo know about it too as he was the reason why I mentioned the other software in the first place :).

**Pull request merged**